topics/contributing/tutorials/running-codespaces/tutorial.md

```diff
@@ -178,30 +178,44 @@ When you have finished your changes, it all looks good in the preview, you want
 
 
 > <hands-on-title>Comitting changes</hands-on-title>
+> Before you can commit your changes you have to create a branch. You have two options to preform this task:
+>  1. **Option 1: via the terminal**
+>    - Hit <kbd>ctrl+c</kbd> if your preview was still running to stop it
+>    - Create a new branch, commit your changes, push changes:
+>
+>    ```bash
+>    git checkout -b fix-title
+>    git commit -m "update tutorial title" topics/introduction/tutorials/galaxy-intro-short/tutorial.md
+>    git push origin fix-title
```

Comment:
if we do it this way, we need to instruct people at the beginning to make a fork of the GTN, and start the codespace from their own fork